### PR TITLE
add module

### PR DESCRIPTION
**Pull request by ghost**
I added my menu

### CODE REVIEW

File: package/mosenos.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                os.system('cls')
+                print("----Goals----")
+
+                goals()
+
+                input("\nPress Enter to continue...")
+
+            case 3:
+                os.system('cls')
+                print("----Comment from teamate----")
```
Comment (alejandro-aleckmcklaiyre) [inline package/mosenos.py]: I will add my own comment here

**Review by alejandro-aleckmcklaiyre — CHANGES_REQUESTED** [review]

File: package/mosenos.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            case 5:
+                os.system('cls')
+                print("----Comment from teamate----")
+                print(" ")
+
+            case 6:
+                os.system('cls')
+                print("----Comment from teamate----")
+                print(" ")
+
```
Comment (gomez-ashleyhermione) [inline package/mosenos.py:66]: I will add my comment here

**Review by gomez-ashleyhermione — CHANGES_REQUESTED** [review]

File: package/mosenos.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            case 4:
+                os.system('cls')
+                print("----Comment from teamate----")
+                print(" ")
+
+            case 5:
+                os.system('cls')
+                print("----Comment from teamate----")
+                print(" ")
+
```
Comment (esparagoza-mikkakette) [inline package/mosenos.py:62]: I'll add my comment here

**Review by esparagoza-mikkakette — CHANGES_REQUESTED** [review]

**Review by alejandro-aleckmcklaiyre — APPROVED** [review]

File: package/mosenos.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                os.system('cls')
+                print("----Comment from teamate 1----")
+                print(" ")
+                print("Hi! that is a very simple and realistic goal. Good luck!")
+                print("- Aleck")
+                
+            case 4:
+                os.system('cls')
+                print("----Comment from teamate----")
+                print(" ")
```
Comment (deleon-jasminerobelle) [inline package/mosenos.py]: I will add my comment here.

**Review by deleon-jasminerobelle — CHANGES_REQUESTED** [review]

**Review by deleon-jasminerobelle — APPROVED** [review]

**Review by esparagoza-mikkakette — APPROVED** [review]

**Review by gomez-ashleyhermione — APPROVED** [review]

**Pull request merged**